style.css

```diff
@@ -0,0 +1,243 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	font-size: 62.5%;
+	background: #393634;
+}
+
+.container {
+	display: flex;
+	color: white;
+	font-size: 1rem;
+	font-family: 'Poppins', sans-serif;
+}
+
+p {
+	line-height: 1.5rem;
+	padding: 1%;
+}
+
+a {
+	text-decoration: none;
+}
+
+h1, h2, h4 {
+	color: #ff7292;
+}
+
+h3 {
+	border-bottom: dotted 0.5px #ff7292;
+	padding-bottom: 2.5%;
+	margin-left: 10%;
+	margin-right: 10%;
+}
+
+span {
+	color: white;
+}
+
+button {
+	border-radius: 100%; 
+	color: #ff7292;;
+	background: #3b3837;
+	border-color: #ff7292;
+	cursor: pointer;	
+}
+
+.menu-image {
+	width: 2.2rem;
+	height: 2.2rem;
+	margin-top: 25%;
+}
+
+.menu-logo {
+	width: 6rem;
+	height: 6rem;
+	margin-top: 30px;
+}
+
+.menu {
+	width: 15%;
+	display: flex;
+	flex-wrap: wrap;
+	border-right: solid black 1px;
+	justify-content: space-between;
+	position: fixed;
+	height: 100vh;
+	background-color: #3b3837;/*#393634*/	
+}
+
+nav {
+	display: flex;
+	flex-wrap: wrap;
+	text-align: center;
+}
+
+nav a {
+	width: 100%;
+	padding-top: 30px;
+	padding-bottom: 30px;
+	border-bottom: solid 0.5px gray;
+	margin-left: 30%;
+	margin-right: 30%;
+	color: white;
+	
+}
+
+nav a:hover, nav a.current {      
+    color: #ff7292;
+	border-color: #ff7292;
+}
+	
+.navigation, .links, .logo {
+	width: 100%;
+}
+
+.links, .logo {
+	display: flex;
+	justify-content: space-around;
+	
+}
+
+.left.left-contact a {
+	color: #ff7292;
+}
+
+.right-section {
+	width: 100%;
+	margin-left: 15%;
+}
+
+.hidden-menu-button {
+	position: absolute;
+	top: 2%;
+	right: 2%;
+	padding: 15px;
+	display: none;
+}
+
+.info {
+	display:flex;
+	justify-content: space-around;
+	padding: 30px;
+	margin: 7%;
+	height: auto;
+	flex-wrap: wrap;
+}
+
+.info.info-portfolio {
+	margin-top: auto;
+}
+
+.info.contact-info {
+	margin-bottom: auto;
+	margin-top: 5%;
+}
+
+.welcome-heading, .welcome-heading2 {
+	font-size: 4rem;
+	font-family: 'Abril Fatface', cursive;
+	text-align: center;
+}
+.welcome-heading2 {
+	font-size: 2rem;
+	padding: 14px;
+}
+
+.left, .right {
+	width: 40%;
+}
+
+.left.project-left {
+	width: 50%;
+}
+
+.avatar { /*contact image can be as it is, tbc*/
+	width: 100%;
+}
+
+.contact, .left.contact-left {
+	text-align: center;
+	color: #ff7292;
+	width: 100%;
+}
+.left.contact-left a {
+	color: #ff7292;
+}
+
+.project-details {
+	text-align: center;
+	width: auto;
+}
+
+.project-img { /*tbc depending on how the project looks*/
+	padding-top: 2%;
+	width: 100%;
+}
+
+.software-used-p {
+	color: #ff7292;
+	font-size: 0.8rem;
+	padding: 5%;
+	
+}
+
+.portfolio-img, .porfolio-heading .avatar {
+	width: 100%;
+	display: flex;
+	justify-content: center;
+}
+
+.porfolio-heading h1 {
+	margin-top: 6%;
+
+}
+
+.portfolio-img img {
+	width: 25%;
+	margin-bottom: 6%;
+}
+
+footer {
+	text-align: center;
+	border-top: solid 1px #ff7292;
+	margin-left: 10%;
+	margin-right: 10%;
+	margin-bottom: 10%;
+	padding: 10px;
+}
+
+.footer-p {
```

Comment:
why `.footer-p`? it wont necessarily always be a p tag. maybe `footer-text` might be better?